02-getting-account-data/index.js

```diff
@@ -0,0 +1,38 @@
+import { SDK } from 'NEWSDKNAME'
+
+// SDK exposes all you need like anchor-link does, users don't need to import from core directly, if they need something they do
+// import { Account } from 'NEWSDKNAME'
+
+const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default plugins (something like if (browser) { plugins = [anchor-browser, atomichub] })
+
+async function main() {
+    const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
+
+    // Loads account
+    const account = await session.accounts[0].get()
```

Comment:
I think a session would only ever have 1 account, so it should probably be:

```js
const account = await session.account.get()
```

Do you imagine a world where you login and multiple accounts are returned?

Reply:
I'm not entirely sure what `session.account` is though, unless it's just an account name? If that's the case, maybe it's something like:

```js
const account = await Account.get(session.account)
```

Reply:
We talked about the new SDK supporting multiple account, but now that I think about it, each account would probably have its own session, right?

Reply:
Correct 👍